client/css/style.css

```diff
@@ -1572,9 +1583,6 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 
 #help .help-item {
 	display: table-row;
-}
-
-#help .help-item {
```

Comment:
This was flagged by my stylelint linter on Atom as duplicate entry. Not sure why my editor reports different things than our actual stylelint configuration...